tasks/configure-Debian.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Create /etc/default/kibana
+  template:
+    src: Ubuntu.default.j2
+    dest: /etc/default/kibana
+    # any files under /etc/default should be mode 644, but the package installs
+    # the file as mode 0664
```

Comment:
* the explanation above should be added as a comment.
* the issue should be reported to upstream if not yet.